server/schema/hod.schema.ts

```diff
@@ -21,7 +21,7 @@ export const hod = pgTable('hod', {
     .notNull(),
   message: text('message'),
   isActive: boolean('is_active').default(true).notNull(),
-  createdOn: date('created_on').defaultNow().notNull(),
+  createdOn: timestamp('created_on').defaultNow().notNull(),
```

Comment:
This will remain a date, we don't know or care what time of the day did a person become an HOD

Reply:
Maybe try this:
```suggestion
  createdOn: date('created_on', { mode: 'date' }).defaultNow().notNull(),
```

Reply:
@aryansri-19 did this work?

Reply:
Yeah, changed this back in the latest push.